lib/ai/model/prompt.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package model
+
+import "fmt"
+
+var observationPrefix = "Observation: "
+var thoughtPrefix = "Thought: "
+
+const PromptSummarizeTitle = `You will be given a message. Create a short summary of that message.
+Respond only with summary, nothing else.`
+
+const InitialAIResponse = `Hey, I'm Teleport - a powerful tool that can assist you in managing your Teleport cluster via OpenAI GPT-4.`
+
+func PromptCharacter(username string) string {
```

Comment:
Does this prompts affect production chats? If yes, have we done A/B tests to check it. Not a blocker for this PR but something that would be good to chase for future.

Reply:
It does, but this is also very similar to the old prompt with an added sentence or so. I have done significant testing on my end to make sure it behaves nicely.